bin/generate-pdf/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "generate-pdf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "generatePdf.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Not sure this matters for a package we're not publishing, but to avoid confusion should we mirror the license line in our main apps package?

https://github.com/code-dot-org/code-dot-org/blob/d88ac114396f93620e9b768c4031fb08c06e7189/apps/package.json#L5

Reply:
👍